mpd/events.go

```diff
@@ -6,13 +6,13 @@ type EventStream struct {
 	XMLName     xml.Name `xml:"EventStream"`
 	SchemeIDURI *string  `xml:"schemeIdUri,attr"`
 	Value       *string  `xml:"value,attr,omitempty"`
-	Timescale   *int64   `xml:"timescale,attr"`
+	Timescale   *uint    `xml:"timescale,attr"`
```

Comment:
why isn't this one a `uint64`?

Reply:
Matching the spec e.g. `<xs:attribute name="timescale" type="xs:unsignedInt/>` though in practicality they are likely the same so can make it `uint64` if you'd like